test/e2e/certificates_test.go

```diff
@@ -354,6 +356,50 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+	
+	Context("OCP-65035-CLOUD_CREDENTIALS_SECRET_NAME works well in mint/passthrough clusters", func() {
+		It("should obtain a valid LetsEncrypt certificate", func() {
+			
+			By("getting Infrastructure object")
+			infra, err := configClient.Infrastructures().Get(ctx, "cluster", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			if infra.Status.PlatformStatus.Type != configv1.GCPPlatformType {	
+				Skip("Skip this case for current plaform, it's not gcp.")
+			}
+
+			By("Check cloud credential in cluster")
+			_, err = loader.KubeClient.CoreV1().Secrets("cert-manager").Get(ctx, "gcp-credentials", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+	
+			By("creating CredentialsRequest object")
+			loader.CreateFromFile(testassets.ReadFile, filepath.Join("testdata", "credentials", "credentialsrequest_gcp.yaml"), "")
+
+			By("Configure cert-manager to use credential, setting this credential secret name in subscription object")
+			credentialValue := "gcp-credentials"
+			err = patchSubscriptionWithCloudCredential(ctx, loader, credentialValue)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("creating new certificate ClusterIssuer")
+			clusterIssuerName := "acme-dns01-clouddns-ambient"
+			loader.CreateFromFile(testassets.ReadFile, filepath.Join("testdata", "acme", "clusterissuer_gcp.yaml"), "")
+			defer certmanagerClient.CertmanagerV1().ClusterIssuers().Delete(ctx, clusterIssuerName, metav1.DeleteOptions{})
+
+			By("creating a test namespace")
+                        ns, err := loader.CreateTestingNS("e2e-acme-ambient-dns01")
+                        Expect(err).NotTo(HaveOccurred())
+                        defer loader.DeleteTestingNS(ns.Name)
```

Comment:
```suggestion
			By("creating a test namespace")
			ns, err := loader.CreateTestingNS("e2e-acme-ambient-dns01")
			Expect(err).NotTo(HaveOccurred())
			defer loader.DeleteTestingNS(ns.Name)
```

just nit: need to align space/tab

Reply:
updated.